tests/installation/partitioning_raid.pm

```diff
@@ -148,45 +148,71 @@ sub run() {
 
     # user defined
     send_key $cmd{custompart};
+    assert_screen 'custompart_option-selected';
     send_key $cmd{next};
     assert_screen 'custompart';
 
     send_key "tab";
-    send_key "down";    # select disks
+    assert_screen 'custompart_systemview-selected';
+    send_key "down";
+    assert_screen 'partitioning_raid-hard_disks-selected';
+
     if (get_var("OFW")) {    ## no RAID /boot partition for ppc
+        wait_idle 3;
+        save_screenshot;
         send_key 'alt-p';
         if (!get_var('UEFI')) {    # partitioning type does not appear when GPT disk used, GPT is default for UEFI
             assert_screen 'partitioning-type';
             send_key 'alt-n';
         }
         assert_screen 'partitioning-size';
         send_key 'ctrl-a';
```

Comment:
I don't know how I ended up there. Disregard. I will take a look again into the current state and comment there.